### ISSUE TITLE
Some implement details wants to confirm

### ISSUE THREAD

**Issue by ShinoSpace** [WXinlong/SOLO#170]
I read through the entire code and found maybe a little different with paper solov2. these question are raised from solov2_head.py. here's my think:
1. in method `split_feats`:

``` python
def split_feats(self, feats):
        return (F.interpolate(feats[0], scale_factor=0.5, mode='bilinear'),
                feats[1],
                feats[2],
                feats[3],
                F.interpolate(feats[4], size=feats[3].shape[-2:], mode='bilinear'))
```

is that means you use FPN features scale from 1/4 to 1/64, and the 1/4, 1/64 feature are down and up to 1/8, 1/32 respectively?

2. in `loss` ->` solov2_target_single`:

``` python
mask_feat_size = ins_pred.size()[-2:]
ins_label_list, cate_label_list, ins_ind_label_list, grid_order_list = multi_apply(
            self.solov2_target_single,
            gt_bbox_list,
            gt_label_list,
            gt_mask_list,
            mask_feat_size=mask_feat_size)
```

i noticed that arg `mask_feat_size` in ` solov2_target_single` is actually the size of unified mask representation which is fused 1/4 feature generated from FPN. Is that correct?

appreciate to your answer!
